R/bind.R

```diff
@@ -14,7 +22,7 @@
 #' dm_1 <- dm_nycflights13()
 #' dm_2 <- dm(mtcars, iris)
 #' dm_bind(dm_1, dm_2)
-dm_bind <- function(..., repair = "check_unique", quiet = FALSE) {
+dm_bind <- function(..., .repair = "check_unique", .quiet = FALSE) {
```

Comment:
Do we expect these dots to be named ?
If not, dotting the arguments might not be necessary unless we want to have them named consistently over all functions.
I can imagine a list of named dm used with `!!!` but I think these would be fed to the dot even if they're named `quiet` or `repair`